### PR TITLE
Bring Navigation out of alpha

### LINKED ISSUE #2558: Remove alpha tag from Navigation component
**Is your feature request related to a problem? Please describe.**
The new version of the Navigation component will be an alpha component.

This component should be amended and tested according to the a11y requirements to remove the tag.

**Describe the solution you'd like**
Remove alpha tag after successful a11y swarm on the component.

**Describe alternatives you've considered**

**Testing notes**

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by tochwill**
Resolves https://github.com/bbc/psammead/issues/2558

**Overall change:** 
Bring the navigation component out of alpha.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.0 | [PR#XXXX](https://github.com/bbc/psammead/pull/XXXX) Bring Navigation out of alpha |
```
Comment (tochwill) [inline packages/components/psammead-navigation/CHANGELOG.md]:
```suggestion
| 6.0.0 | [PR#3028](https://github.com/bbc/psammead/pull/3028) Bring Navigation out of alpha |
```

File: packages/components/psammead-navigation/README.md [hunk 2]
```diff
@@ -166,7 +162,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
       Sport
     </DropdownLi>
   </DropdownUl>
-</CanonicalDropdown>
+</CanonicalDropdown>;
```
Comment (tochwill) [inline packages/components/psammead-navigation/README.md]:
```suggestion
</CanonicalDropdown>
```

File: packages/components/psammead-navigation/README.md [hunk 3]
```diff
@@ -184,7 +180,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
     console.log('Handle onClick action');
   }}
   script={latin}
-/>
+/>;
```
Comment (tochwill) [inline packages/components/psammead-navigation/README.md]:
```suggestion
/>
```

File: packages/components/psammead-navigation/README.md [hunk 4]
```diff
@@ -199,7 +195,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
   onToggle="menu.toggleVisibility"
   script={latin}
   dir={dir}
-/>
+/>;
```
Comment (tochwill) [inline packages/components/psammead-navigation/README.md]:
```suggestion
/>
```

**Review by AlistairGempf — APPROVED** [review]
🎉  🙌

**Review by OlgaLyubin — COMMENTED** [review]
`publishConfig tag alpha` should be deleted from `package.json`

**Review by DenisHdz — CHANGES_REQUESTED** [review]
You have to also remove the following from the `package.json`:

```
  "publishConfig": {
    "tag": "alpha"
  }
```

**Review by chris-hinds — APPROVED** [review]
👍

**Review by DenisHdz — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Review by OlgaLyubin — APPROVED** [review]
👍

**Pull request merged**